l10n_br_account_nfe/models/account_move_line.py

```diff
@@ -31,3 +31,8 @@ def write(self, values):
                 invoice.fiscal_document_id.action_document_confirm()
                 invoice.fiscal_document_id._document_export()
         return result
+
+    @api.onchange('city_taxation_code_id')
+    def _onchange_city_taxation_code_id(self):
```

Comment:
Este onchange já existe no fiscal.. acredito que se for fazer algo deveria ser aqui.

https://github.com/OCA/l10n-brazil/blob/0bd8e00ea39beab3dd09fc4baea13ddd51990e92/l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py#L833